src/status_im/contexts/wallet/common/utils_test.cljs

```diff
@@ -106,3 +106,20 @@
 
       (is (= (utils/get-wallet-qr wallet-singlechain)
              "x000")))))
+
+(deftest test-calculate-fiat-change
+  (testing "calculate-fiat-change function"
+    (is (money/equal-to (utils/calculate-fiat-change 38.23 1.34) 0.512282))
+    (is (money/equal-to (utils/calculate-fiat-change 45234 2.89) 1307.2626))
+    (is (money/equal-to (utils/calculate-fiat-change 23 0) 0))
+    (is (money/equal-to (utils/calculate-fiat-change 45 nil) 0))
+    (is (money/equal-to (utils/calculate-fiat-change 98 "") 0))))
+
+(deftest test-prettify-percentage-change
+  (testing "prettify-percentage-change function"
+    (is (= (utils/prettify-percentage-change nil) "0.00"))
+    (is (= (utils/prettify-percentage-change "") "0.00"))
+    (is (= (utils/prettify-percentage-change 0.5) "0.50"))
+    (is (= (utils/prettify-percentage-change 1.113454) "1.11"))
+    (is (= (utils/prettify-percentage-change -0.35) "0.35"))
+    (is (= (utils/prettify-percentage-change -0.78234) "0.78"))))
```

Comment:
Shouldn't this be `"-0.78"?

Reply:
@ulisesmac - We need to display the absolute value (unsigned number) in the UI.

<img width="476" src="https://github.com/status-im/status-mobile/assets/19339952/964f5d05-3e2b-4bbc-8032-fb5f97abd153">

If the value is negative (loss), we display it in red color. If it's positive (profit), we display it in green color.

Reply:
Thanks for the explanation! :)